src/Tizen.Wearable.CircularUI.Chart.Forms/PieDataItem.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xamarin.Forms;
+
+namespace Tizen.Wearable.CircularUI.Chart.Forms
+{
+    /// <summary>
+    /// PieDataItem is an inheritance type of DataItem for the PieChartView..
+    /// </summary>
+    public class PieDataItem : IDataItem
+    {
+        public PieDataItem()
+        {
+        }
+
+        public PieDataItem(TextItem label, double value, Color color)
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets a value.
+        /// </summary>
+        public double Value { get; set; }
+
+        /// <summary>
+        /// Gets or sets a color of each DataItem.
+        /// If DataItem.Color is not set, The color of Slice is set with designated color.
+        /// </summary>
+        public Color Color { get; set; } = Color.Default;
+
+        /// <summary>
+        /// Gets or sets a value text of DataItem.
+        /// </summary>
+        public TextItem ValueText { get; set; }
```

Comment:
`ValueText` was added previous PR, but it seems strange, It is not readable

Reply:
Is there a name you could suggest ?

Reply:
I think, it should be `Label`

Reply:
removed PieDataItem. all Property of ValueLabel moved into PieChartView.